index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Intuit Accounts - Sign Up</title>
+    <link rel="stylesheet" href="stylesheets/main.css" />
+  </head>
+  <body>
+    <header id="main-header" class="flex-h">
+      <div class="nav-container flex-h align-center">
+        <div class="m-r-1">
+          <p>Already have an Intuit account?</p>
+          <a id="forgot-id-pass" href="#">I forgot my User ID or Password</a>
+        </div>
+        <div class="m-r-2">
+          <button id="sign-in-button" class="btn">
+            <i class="fas fa-lock"></i>
+            Sign In
+          </button>
+        </div>
+      </div>
+    </header>
+    <!--End of Header Section-->
+
+    <div id="logo" class="flex-v align-center m-t-2">
+      <header>
+        <a href=""><img src="./images/intuit.png" alt="intuit"/></a>
+      </header>
+
+      <footer>
+        <div class="flex-h">
+          <a class="flex-h align-center m-r-2" href=""
+            ><img src="./images/mint.png" alt="mint" /><span>mint</span></a
+          >
+          <a class="flex-h align-center m-r-2" href=""
+            ><img src="./images/quickbooks.png" alt="quickbooks" /><span
+              >quickbooks</span
+            ></a
+          >
+          <a class="flex-h align-center m-r-2" href=""
+            ><img src="./images/turbotax.png" alt="turbotax" /><span
+              >turbotax</span
+            ></a
+          >
+        </div>
+      </footer>
+    </div>
+    <!--End of Logo Section-->
+
+    <main id="signup-form" class="flex-v align-center m-t-1">
```

Comment:
- [ ] Your page's form is a little less wide than the original one. This makes your page look slightly different because some elements display in two lines when they should be in just one, or have to scroll down more to see all the elements. Please make the necessary adjustments.